src/main/rx/rx.h

```diff
@@ -117,7 +121,7 @@ PG_DECLARE_ARRAY(rxChannelRangeConfig_t, NON_AUX_CHANNEL_COUNT, rxChannelRangeCo
 typedef struct rxConfig_s {
     uint8_t rcmap[RX_MAPPABLE_CHANNEL_COUNT];  // mapping of radio channels to internal RPYTA+ order
     uint8_t serialrx_provider;              // type of UART-based receiver (0 = spek 10, 1 = spek 11, 2 = sbus). Must be enabled by FEATURE_RX_SERIAL first.
-    uint8_t sbus_inversion;                 // default sbus (Futaba, FrSKY) is inverted. Support for uninverted OpenLRS (and modified FrSKY) receivers.
+    uint8_t frsky_inversion;                 // default sbus (Futaba, FrSKY) and FPort (FrSky) is inverted. Support for uninverted OpenLRS (and modified FrSKY) receivers.
```

Comment:
Why has this been renamed? It seems to me that it applies to sbus, not frsky.

And even if renaming is desired, could it should be done in a separate PR, having the rename in this PR introduces a lot of noise and makes this PR harder to review.

Reply:
It was renamed to account for the fact that the setting is now used for SBus and FPort (as the comment states). I am not convinced that the new name is the most appropriate, but it's the best I could come up with, so suggestions are welcome. Will open a separate PR.